muhasin/index.js

```diff
@@ -0,0 +1,94 @@
+// 3.Declare the variables a, b and c, and give them number values. Check which one out of the three variables has the largest value and print it to the console. Then change the values of the variables to see if your conditional still works.
+
+// 1. Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+let num1 = 5;
+let num2 = 80;
+
+if ((num1 >= 50 && num1 <= 99) || (num2 >= 50 && num2 <= 99)) {
+  console.log(true);
+} else {
+  console.log(false); // false
+}
+
+// 2. Check if three given integer values are in the range 50 to 99 (inclusive). Print true if one or more of them are in the range.
+let int = 40;
+let int2 = 50;
+let int3 = 60;
+
+if (
+  (int >= 50 && int <= 99) ||
+  (int2 >= 50 && int2 <= 99) ||
+  (int3 >= 50 && int3 <= 99)
+) {
+  console.log(true); // true
+} else {
+  console.log(false);
+}
+
+// 3. Declare the variables a, b and c, and give them number values. Check which one out of the three variables has the largest value and print it to the console. Then change the values of the variables to see if your conditional still works.
+
+let a = 80;
+let b = 90;
+let c = 100;
+if (a >= b && a >= c) {
+  console.log(a);
+} else if (b >= a && b >= c) {
+  console.log(b);
+} else {
+  console.log(c);
+}
+
+// 4. Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+let str1 = "thon";
+let str2 = "Py" + str1;
+if (str2.startsWith("Py")) {
+  console.log(str1); // thon
+}
+
+// 5. Calculate the sum of the two integers. If the sum is in the range 50 to 80, print 65, otherwise print 80.
+let int4 = 15;
+let int5 = 5;
+let sum = int4 + int5;
+console.log("sum ==>", sum); // 20
+if (sum >= 50 && sum <= 80) {
+  console.log(65);
+} else {
+  console.log(80); // 80
+}
+
+// 6. Check whether the sum of two integers is 8, or whether their difference is 8. If one of these is the case, print true.
+let firstInt = 4;
+let secondInt = 4;
+if (firstInt + secondInt == 8 || firstInt - secondInt == 8) {
```

Comment:
Task6: The answer is correct but its better to use Math.abs() to find out the difference between two values.